### PR TITLE
v2

### PR DESCRIPTION
**Pull request by lishaduck**
This PR is for @ParkerH27 and suggests merging next into [`main`](https://github.com/PSDTools/PHS-Map/tree/main) when he's ready.

```[tasklist]
### Features
- [ ] #2
- [ ] #4
```

### CODE REVIEW

**Comment by lishaduck** [comment]
Ought [`data/*.txt`](https://github.com/PSDTools/PHS-Map/tree/f320299c67361ecf739504f345b483f43876a60b/data) be [`data/*.json`](https://github.com/PSDTools/PHS-Map/tree/next/data)?

**Comment by lishaduck** [comment]
Hey, I fixed the deployments, @ParkerH27.

**Comment by lishaduck** [comment]
Is this ready to merge, @ParkerH27?

File: One-time-use.html [hunk 1]
```diff
@@ -1,156 +1,197 @@
...
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Lato"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script
+      type="text/javascript"
+      src="pathing/pathfinding-browser.js"
```
Comment (lishaduck) [inline One-time-use.html:17]: This needs to get updated.

File: One-time-use.html [hunk 2]
```diff
@@ -1,156 +1,197 @@
...
+      href="https://www.psdr3.org/images/logos/psd.png"
+      rel="shortcut icon"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://www.w3schools.com/lib/w3-theme-black.css"
+    />
+    <script type="text/javascript" src="data/Level0.js"></script>
+    <script type="text/javascript" src="data/Level1.js"></script>
```
Comment (lishaduck) [inline One-time-use.html:29]: This file doesn't exist, @ParkerH27.

File: One-time-use.html [hunk 3]
```diff
@@ -1,156 +1,197 @@
...
+            pageLanguage: "en",
+            layout: google.translate.TranslateElement.InlineLayout.SIMPLE,
+          },
+          "google_translate_element"
+        );
+      }
+      // ]]>
+    </script>
+    <script
+      src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"
```
Comment (lishaduck) [inline One-time-use.html:150]: This makes a lot of issues in the GPA Calculator, @ParkerH27. Do you want to remove it?

File: One-time-use.html
Comment (lishaduck) [inline One-time-use.html:1]: Should this be `one-time-use/index.html`, @ParkerH27?

Comment (ParkerH27) [reply One-time-use.html:1]: i dont know or really care.... the only thing is if you change the file name you have to change the link to it for the help button

File: One-time-use.html (hunk 3 above)
Comment (ParkerH27) [reply One-time-use.html:150]: if thats referring to the translate element then no... It at least needs to be available online...

**Review by ParkerH27 — APPROVED** [review]

**Comment by lishaduck** [comment]
Is the `data/`folder still needed?

File: One-time-use.html (hunk 1 above)
Comment (lishaduck) [reply One-time-use.html]: Just deleting it.

**Comment by lishaduck** [comment]
Well, looks like 6ee2b89 broke somethin'. Ooops.

**Comment by lishaduck** [comment]
The regressions from 6ee2b8998424a76d165b463c96f26d63e083792f and 
03b2f824db00a7bd86ea5840c9229d324f2482dd have been fixed, and `data/*.txt` has successfully been moved. Thus, I'm merging this.

**Pull request merged**